### ISSUE TITLE
not a single example!

### ISSUE THREAD

**Issue by John-Blair** [MicrosoftDocs/azure-devops-docs#6159]
Why not be really helpful and include a few examples of how to use this command. There are lots of parameters with vague usage details. I want to copy a folder from a Test app service to a Live app service - maybe start with that example? Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 622daa35-c6a1-dac3-17ee-3f9308fc60cd
* Version Independent ID: 2c28a59b-8ef5-d7ab-46a6-58748ec552f5
* Content: [Azure File Copy task - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-file-copy?view=azure-devops)
* Content Source: [docs/pipelines/tasks/deploy/azure-file-copy.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/tasks/deploy/azure-file-copy.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @RoopeshNair
* Microsoft Alias: **ronai**

**Comment by msebolt** [comment]
@John-Blair  Thank you for the feedback, assigning this to the author for review.

**Comment by LaiHoangLe** [comment]
I help the same problem. It is so hard to follow because no example is provided

**Comment by goyomiller** [comment]
I agree as well, there are so many nuances to many of these azure tasks, one can spend many hours guessing parameters and their expected values.

**Comment by arcotek-ltd** [comment]
I agree with OP and as highlighted in #3290 the table of attributes does not match the schema.
For example:
![image](https://user-images.githubusercontent.com/26305145/80008002-fc506000-84be-11ea-820b-4c7a902a0922.png)

Where is `storage:` in the attributes list?

This documentation seriously needs improving.

Thanks

**Comment by arcotek-ltd** [comment]
It would also help to highlight the fact that tasks (except PowerShell) do not work with service connectors that are scoped to management groups.

It would have saved me ~6 hours working out why it was missing the subscription ID and ResourceGroupName from the upload URL!
https://developercommunity.visualstudio.com/idea/1002844/service-connection-work-with-all-tasks-at-the-mana.html

**Comment by cxwtool** [comment]
This issue hasn't been updated in more than 180 days, so we've closed it. If you feel the issue is still relevant and needs fixed, please reopen it and we'll take another look. We appreciate your feedback and apologize for any inconvenience.
